test/cpp/end2end/xds/xds_override_host_end2end_test.cc

```diff
@@ -267,6 +316,140 @@ TEST_P(OverrideHostTest, DrainingExcludedFromOverrideSet) {
   EXPECT_EQ(2, backends_[2]->backend_service()->request_count());
   ResetBackendCounters();
 }
+
+TEST_P(OverrideHostTest, HostOverridden) {
+  CreateAndStartBackends(4);
```

Comment:
We probably only need 3 backends for this test, 1 in the first cluster and 2 in the second.